portal-web/test/functional/com/liferay/portalweb/macros/RemoteApps.macro

```diff
@@ -254,6 +314,78 @@ definition {
 		RemoteApps.viewEmptyRemoteTable();
 	}
 
+	macro ViewDisabledCustomElementFields {
+		AssertNotEditable(
+			custom_entryName = "${entryName}",
+			key_text = "Name",
+			locator1 = "TextInput#ANY",
+			value1 = "${entryName}");
+
+		if (isSet(entryDescription)) {
+			AssertNotEditable(
+				entryDescription = "${entryDescription}",
+				key_text = "_description",
+				locator1 = "TextArea#ANY",
+				value1 = "${entryDescription}");
+		}
+
+		if (isSet(entrySourceCodeURL)) {
+			AssertNotEditable(
+				entrySourceCodeURL = "${entrySourceCodeURL}",
+				key_text = " Source Code URL ",
+				locator1 = "TextInput#ANY_SPECIFIC",
+				value1 = "${entrySourceCodeURL}");
+		}
+
+		if (isSet(entryType)) {
+			AssertNotEditable(
+				entryType = "${entryType}",
+				key_id = "ClientExtensionAdminPortlet_typeLabel",
+				locator1 = "RemoteAppsEntry#URL",
+				value1 = "${entryType}");
+		}
+
+		AssertNotEditable(
+			entryCssurlText = "${entryCssurlText}",
+			key_text = "CSS URL",
+			locator1 = "TextArea#ANY",
+			value1 = "${entryCssurlText}");
+
+		if (isSet(entryFriendlyElementName)) {
+			AssertNotEditable(
+				entryFriendlyElementName = "${entryFriendlyElementName}",
+				key_id = "ClientExtensionAdminPortlet_Friendly-URL-Mapping",
+				locator1 = "RemoteAppsEntry#URL",
+				value1 = "${entryFriendlyElementName}");
+		}
+
+		AssertNotEditable(
+			entryHtmlName = "${entryHtmlName}",
+			key_text = "HTML Element Name",
+			locator1 = "TextInput#ANY",
+			value1 = "${entryHtmlName}");
+
+		if (isSet(entryCategoryName)) {
+			AssertNotEditable(
+				entryCategoryName = "${entryCategoryName}",
+				key_text = "Portlet Category Name",
+				locator1 = "TextInput#ANY",
+				value1 = "${entryCategoryName}");
+		}
+
+		AssertNotEditable(
+			entryURLText = "${entryURLText}",
+			key_text = "ClientExtensionAdminPortlet_URLs",
+			locator1 = "TextArea#ANY",
+			value1 = "${entryURLText}");
+
+		AssertNotEditable(
+			entryProperties = "${entryProperties}",
+			key_text = "properties",
+			locator1 = "TextArea#ANY",
+			value1 = "${entryProperties}");
+	}
+
```

Comment:
same as previous comment. 

Also, avoid using the term `View` to "Assert" or something else. Be more specific with what this macro is supposed to do. I know in RemoteApps.macro file there's other macros with `view` but we need to stop doing this.

Reply:
Done!